package.json

```diff
@@ -22,6 +22,7 @@
     "./packages/*"
   ],
   "scripts": {
+    "prune:deps": "find . -name 'node_modules' -type d -prune -exec rm -rf '{}' +",
```

Comment:
That's just a helper command to remove all `node_modules` directories nested inside workspaces, we can either remove or keep it if that could somehow be useful later

Reply:
Good

Reply:
Does this command work on windows as well?